### ISSUE TITLE
HTTPS

### ISSUE THREAD

**Issue by anbu-squad** [UnamSanctam/SilentCryptoMiner#2258]
I watched the traffic through Wireshark
and there post requests to the panel are easy to read

it's probably worth using the protocol https?

**Comment by UnamSanctam** [comment]
Yes using https is preferred if you can, if you have a domain then you can get a free certificate for your website at https://letsencrypt.org/ or if you only have an IP address then you can use https://zerossl.com/.

**Comment by anbu-squad** [comment]
the panels started to block, as I understand it
For me personally, my domain no longer works

and then I thought I could try this

https://panelsite
http://panelsite
tor://panelsite

sequence of protocol changes, relatively speaking
if the certificate will not work
switch to HTTP
if domains are blocked
last one left TOR

**Comment by UnamSanctam** [comment]
Your domain likely got reported and taken down, eventually it will always happen when using a domain due to how the internet works in regards to domains. Http and https are really the same except that https is secure (traffic is encrypted) so that won't really make a difference in regards to a domain being reported (except that https might make it last longer). Tor would be preferable but to use that a local Tor proxy needs to be installed on the computer and then the traffic needs to be proxied through it (by default port 9000 I believe). There's also the issue with Tor connections commonly being very slow due to it having to go through several servers and with the endpoints being provided by volunteers.

**Comment by anbu-squad** [comment]
It’s probably worth preparing several backup domains

perhaps backup domains will not be blocked if they do not receive suspicious traffic
although this is just my guess

**Comment by UnamSanctam** [comment]
Yes backups can be good, the domains usually get taken down due to things like VirusTotal and other sandboxes, when they sniff the traffic it gets reported and then eventually it might get taken down.
